pkgs/development/tools/vala-language-server/default.nix

```
@@ -15,13 +15,13 @@
 
 stdenv.mkDerivation rec {
   pname = "vala-language-server";
-  version = "0.48.4";
+  version = "0.48.5";
 
   src = fetchFromGitHub {
     owner = "Prince781";
```

```suggestion
    owner = "vala-lang";
```

Please also update homepage.

---

Thanks